index.html

```diff
@@ -0,0 +1,27 @@
+<html ng-app="angularBooks">
+<head>
+  <title>Books App</title>
+
+  <!-- STYLESHEETS -->
+  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+  <link rel="stylesheet" href="/public/styles/main.css">
+
+  <!-- VENDOR SCRIPTS -->
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.6/angular.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.2/angular-route.min.js"></script>
+
+  <!-- APPLICATION SCRIPTS -->
+  <script src="/public/scripts/app.js"></script>
+  <script src="/public/scripts/controllers/BooksIndexController.js"></script>
+  <script src="/public/scripts/controllers/BooksShowController.js"></script>
+
+</head>
+<body>
+  <div class="jumbotron">
+    <div class="container">
```

Comment:
Why do you have the jumbotron and container? Is it necessary, or just copy-pasted from other assignments?